bin/zowe-setup-certificates-prep.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+sed \
```

Comment:
I think you need to also overwrite the default keystore directory with the one from the yaml?